plan/validator.go

```diff
@@ -353,3 +436,14 @@ func checkFieldLengthLimitation(colDef *ast.ColumnDef) error {
 	}
 	return nil
 }
+
+// See https://dev.mysql.com/doc/refman/5.7/en/identifiers.html
+func isIncorrectName(name string) bool {
```

Comment:
All statements have been covered by test. I think we can add a single unit test if this function become more complex in future.